### ISSUE TITLE
Unclear Copyright/License

### ISSUE THREAD

**Issue by joernneumeyer** [Penguin-Library/penguin#68]
## Origin
I recently built [my own website](https://github.com/joernneumeyer/joern-neumeyer.de), which uses Sierra Library.

## Issue
1. Since Sierra Library is Licensed under GPL-2.0, copyright notices have to be applied, but  no unambiguous license and copyright notice was present in any file header
1. It is not clear, whether the GPL-2.0 variant GPL-2.0-only, or GPL-2.0-or-later is intended

## Solution
1. Add a license and copyright notice the the main source file (like `src/index.scss`)
1. Add the intended license code to the license name for clarification

**Comment by maraisr** [comment]
Fair point. :100: @JoanClaret do you think we should explore MIT'ing this?

@joernneumeyer do you think you can submit us a PR, to implement your solution? Would help us a great deal 👏 :bowtie:

**Comment by joernneumeyer** [comment]
PR #69 has been opened. That PR introduces license headers for the GPL-2.0-or-later and copyright notices for `Joan Claret and contributors`.
Personally I would be against switching the license, as I chose Sierra for its simplicity and the GPL, as mentioned.

**Comment by JoanClaret** [comment]
so, if I correctly understood, ` GPL` licences force the users to explicitly indicate the license and copyright notice in their sites. Just like @joernneumeyer did in his site. (thanks for using sierra btw! 😃 ), While MIT licences don't.

then, i would bet for moving to MIT, so more people will be able to use this library. 

opinions? @elboletaire @maraisr

**Comment by JoanClaret** [comment]
more info: https://choosealicense.com/licenses/

**Comment by elboletaire** [comment]
Another reason to consider changing the license is the usage other companies will do of it. With GNU/GPL you force others to specify, somewhere in their code/program, the license of your licensed work, whilst using MIT this is not required.

For that reason, using a MIT license probably will "open more doors".

What I'm not really sure is if it really is required to add the entire GNU license in each file. It's fine to just have a `@license` in files if the license file is in the root path of the project, instead of adding the entire license to each file, which is too much verbose IMHO.

**Comment by joernneumeyer** [comment]
@JoanClaret any GPL licensed code/project requires the user of such code/project to apply the same license to their project (or higher versions of the license, if specified) if they are going to (re-)distribute their added/dependent code/project.
Therefore, enabling the whole idea of [copyleft](https://www.gnu.org/licenses/copyleft.en.html) and preventing proprietary (closed) use of the code/project you wrote.

@elboletaire the file headers proposed in my PR where the recommended way as shown in [this post](https://www.gnu.org/licenses/gpl-howto.html) of the GNU project.
That way, everybody who may want to use code from the licensed project can always see the license of the code, that license's compatibility, the rights they were granted, and the disclaimer of warranty.

**Comment by sburris0** [comment]
Hi, can I get an update on the license change status?

**Comment by juliendude** [comment]
hmmm... interesting thread. It's nice to see an update of Sierra.
But I agree that it limits a lot its usage by dev and companies since with this license.
Which is too bad because Sierra is well done with a small footprint.

@JoanClaret Any update on the change of license by any chance?

**Comment by sburris0** [comment]
ISC is basically updated MIT and should be preferred
